lib/palindrome_check.rb

```diff
@@ -1,7 +1,24 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: linear, O(n) - where n is the length of the input string
```

Comment:
Tiny nit: `/ 2` is unnecessary here - `O(n/2)` simplifies to `O(n)`

:+1: to clarifying what `n` stands for though!